docs/service_spec.md

```diff
@@ -249,6 +255,9 @@ A `POST` request indicates that the test harness wants to start an instance of t
       used by the SDK to verify the peer's certificate chain. The file will contain one or more PEM-encoded x509 
       certificates. The root of the certificate chain presented to the SDK during the TLS handshake must be signed by 
       one of these CA certificates.
+  * `wrapper` (object, optional): If specified contains wrapper configuration.
+    * `wrapperName`: The name of the wrapper.
+    * `wrapperVersion`: The version of the wrapper.
```

Comment:
`wrapper.wrapperName` is redundant. Should we rename to `wrapper.name` and `wrapper.version`?

Reply:
Yeah, that seems reasonable.